Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM jruby:9.2.5.0-jdk-alpine
-RUN apk add --no-cache --update build-base postgresql-dev nodejs curl
-RUN mkdir /app
+
 WORKDIR /app
+
 COPY lib lib
-RUN ./lib/upvs/compile
-RUN gem install bundler
-RUN gem update --system
 COPY Gemfile Gemfile.lock ./
-RUN bundle install --without development:test --path vendor/bundle --deployment
 COPY . .
+
+RUN apk add --no-cache --update build-base postgresql-dev nodejs curl \
```

Comment:
Napadu chapem, ale prave naopak takto ako to mame to je optimized na rychlost deploymentu/buildu cez dokku, lebo upvs/compile trva naozaj dlho a vacsinu casu sa to vobec neupdatuje. preto su oddelene tie casti od seba, aby to boli separe stiahnutelne layers v docker file.